repos/fdbt-site/src/pages/capConfirmation.tsx

```diff
@@ -233,6 +247,13 @@ export const getServerSideProps = async (ctx: NextPageContextWithSession): Promi
                 : 'Rolling days'
             : '';
 
+    const additionalPricing =
+        typeOfCapAttribute.typeOfCap === 'byDistance'
```

Comment:
do we need to check the type of cap? I think we can just see if the additionalPricingAttribute is there and use the presence of that?

Reply:
I don't think this can be done as the attribute is optional

Reply:
^ incorrect